database/entity/0002-add_settings/User.ts

```diff
@@ -1,5 +1,5 @@
 import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, OneToMany } from 'typeorm'
-import { UserSetting } from '../UserSetting'
+import { UserSetting } from '../0002-add_settings/UserSetting'
```

Comment:
Because I deleted the `UserSetting.ts` file in the entity folder I changed the reference here and in all the other similar cases.

Is that correct @Mogge ? Or shall it stay undeleted for ever?
To leave it undeleted would be confusing for our future development I think …